algebird-spark/src/main/scala/com/twitter/algebird/spark/package.scala

```diff
@@ -0,0 +1,46 @@
+package com.twitter.algebird
+
+import org.apache.spark.rdd.{ RDD, PairRDDFunctions }
+
+import scala.reflect.ClassTag
+import com.twitter.algebird._
+
+/**
+ * To use this, you probably want:
+ * import com.twitter.algebird.spark._
+ */
+package object spark {
+  /**
+   * This adds methods to Spark RDDs to use Algebird
+   */
+  implicit class AlgebirdRDD[T](val rdd: RDD[T]) extends AnyVal {
+    /**
+     * Apply an Aggregator to return a single value for the whole RDD
+     */
+    def aggregatorOnAll[B: ClassTag, C](agg: Aggregator[T, B, C]): C =
```

Comment:
There is already `aggregateByKey`. I will add tests and see if the implicit enrichment can differentiate based on the arguments. It may, or it may give the user a very confusing message. Let's see.